feature-libs/user/profile/components/new-user-profile/new-profile.component.spec.ts

```diff
@@ -0,0 +1,169 @@
+import { CommonModule } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  DebugElement,
+} from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import {
+  UntypedFormControl,
+  UntypedFormGroup,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import { By } from '@angular/platform-browser';
+import { RouterTestingModule } from '@angular/router/testing';
+import { NgSelectModule } from '@ng-select/ng-select';
+import {
+  FeaturesConfig,
+  FeaturesConfigModule,
+  I18nTestingModule,
+} from '@spartacus/core';
+import { FormErrorsModule } from '@spartacus/storefront';
+import { UrlTestingModule } from 'projects/core/src/routing/configurable-routes/url-translation/testing/url-testing.module';
+import { BehaviorSubject, Subject, of } from 'rxjs';
+import { NewProfileComponentService } from './new-profile-component.service';
+import { NewProfileComponent } from './new-profile.component';
+import createSpy = jasmine.createSpy;
+@Component({
+  selector: 'cx-spinner',
+  template: ` <div>spinner</div> `,
+})
+class MockCxSpinnerComponent {}
+
+const isBusySubject = new BehaviorSubject(false);
+
+class MockProfileService implements Partial<NewProfileComponentService> {
+  user$ = of({});
+  titles$ = of([]);
+  updateSucceed$ = new Subject<boolean>();
+  form: UntypedFormGroup = new UntypedFormGroup({
+    customerId: new UntypedFormControl(),
+    titleCode: new UntypedFormControl(),
+    firstName: new UntypedFormControl(),
+    lastName: new UntypedFormControl(),
+  });
+  isUpdating$ = isBusySubject;
+  updateProfile = createSpy().and.stub();
+}
+
+describe('NewProfileComponent', () => {
+  let component: NewProfileComponent;
+  let fixture: ComponentFixture<NewProfileComponent>;
+  let el: DebugElement;
+
+  let service: NewProfileComponentService;
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        imports: [
+          CommonModule,
+          ReactiveFormsModule,
+          I18nTestingModule,
+          FormErrorsModule,
+          RouterTestingModule,
+          UrlTestingModule,
+          NgSelectModule,
+          FeaturesConfigModule,
+        ],
+        declarations: [NewProfileComponent, MockCxSpinnerComponent],
+        providers: [
+          {
+            provide: NewProfileComponentService,
+            useClass: MockProfileService,
+          },
+          {
+            provide: FeaturesConfig,
+            useValue: {
+              features: { level: '5.2' },
```

Comment:
maybe config it as latest or * ?

Reply:
Ah, Just cp from somewhere. Will delete it